auraed/src/cells/cell_service/cells/cells.rs

```diff
@@ -243,6 +247,25 @@ impl CellsCache for Cells {
     fn broadcast_kill(&mut self) {
         self.broadcast_kill()
     }
+
+    fn cell_graph(&mut self, node: GraphNode) -> Result<GraphNode> {
+        let (valid, invalid): (Vec<_>, Vec<_>) = self
+            .cache
+            .values_mut()
+            .map(|c| c.cell_graph(node.clone()))
+            .partition_map(|r| match r {
+                Ok(n) => Either::Left(n),
+                Err(e) => Either::Right(e),
+            });
+
+        for node in invalid {
+            if let CellsError::CellNotAllocated { cell_name } = node {
+                _ = self.cache.remove(&cell_name);
```

Comment:
I didn't even know you can have the `_ = ...` without let. Can we put the let in? My IDE complains.